docs/src/clients.md

```diff
@@ -66,7 +66,14 @@ If you are using Kinesis, then in addition to the configuration file entries abo
 Kinesis dispatcher can be configured with other [KPL properties](https://github.com/awslabs/amazon-kinesis-producer/blob/master/java/amazon-kinesis-producer-sample/default_config.properties) in addition to `Region` by including them in the same part of the config file where `Region` is set.
 
 ## Metrics
-Coming soon.
+
+The client has a stand-alone metrics interface, that is based on [micrometer][micrometer]'s api.  Supplied with the base library, is a `NoopMetricsRegistry` that collects nothing as expected.
```

Comment:
Not sure what [micrometer][micrometer] is going to do here; did you mean to have a URL in the second set of brackets, and for those brackets to be parentheses?

Reply:
I think that wants to be `"[Micrometer's API](https://github.com/micrometer-metrics/micrometer)."`

Reply:
Just noticed you merged this. I'll go in and fix it.

Reply:
I'm using reference style links, as they are easier to maintain and make the prose easier to read later on for updates.

See: https://daringfireball.net/projects/markdown/basics
> Markdown supports two styles for creating links: inline and reference. With both styles, you use square brackets to delimit the text you want to turn into a link.